rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Install shade and its dependencies
+  pip:
+    name: "{{ item }}"
+    state: latest
+    extra_args: "--isolated"
```

Comment:
``--isolated`` should not be required because the repo build process will pick up the support_utility_pip_packages list and will ensure those wheels are built. In fact it *must not* be used because this will result in a different version of shade being used any time you execute this task.